Models/Tip.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SmartSaver.Models
+{
+    class Tip
+    {
+        private string[] tipNames;
+        private string[] tipLinks;
```

Comment:
How can Object Tip (singular) contain multiple names?
Instead this class (describing Tip (singular)) should be given properties: `name`, `link`
Also can be links if the same tip is described in multiple sources. 

And then the whole Tip class will be drastically simplified.